src/tutorials/0008-introduction-to-libp2p/04.md

```diff
@@ -0,0 +1,21 @@
+---
+    title: "libp2p in the OSI Model"
+    type: "multiple-choice"
+---
+
+So where does libp2p fit into the current web?
+Usually, we look at the **OSI Model (Open Systems Interconnection Model)** to understand which layer a protocol tries to tackle.
+
+<div class="flex justify-center mv4">
+    <img src="/tutorial-assets/T0008L04-osi-model.png">
+</div>
+
+But the truth is the `OSI Model` as well as the `TCP/IP Model` are just conceptual models. The actual implementations in the current web do not follow these models exactly, but instead it's a mix, i.e., the `TCP/IP Protocol Suite`.
+
+In the end, these models have some shortcomings:
+
+- Actions are repeated through multiple layers (repeated work) because data can go through multiplexing multiple times, or authentication multiple times or even discovery multiple times - it's very inefficient.
```

Comment:
I can make guesses about authentication and discovery because they're normal words, but wouldn't know what multiplexing is.

Reply:
We can remove `multiplexing` in here then?

Reply:
done: 6152555b3f7bfbc1830ab419446ae71b57ecdf05